test/unit/test/streaming/streaming.controllers.CmcdController.js

```diff
@@ -540,39 +540,34 @@ describe('CmcdController', function () {
                             type: MEDIA_SEMGENT_REQUEST_TYPE,
                         };
                         headers = cmcdController.getHeaderParameters(request);
-                        expect(headers).to.have.property(OBJECT_HEADER_NAME);
```

Comment:
I'm not sure what changed to cause these lines to fail. We the tests run, `CmcdModel`'s `_getCmcdDataForMediaSegment` function is not being passed a `mediaType` argument.